webapp/components/admin_console/legal_and_support_settings.jsx

```diff
@@ -96,7 +96,7 @@ export default class LegalAndSupportSettings extends AdminSettings {
                     helpText={
                         <FormattedMessage
                             id='admin.support.aboutDesc'
-                            defaultMessage='Link to About page for more information on your Mattermost deployment, for example its purpose and audience within your organization. Defaults to Mattermost information page.'
+                            defaultMessage='Link to About page for more information on your Mattermost deployment, for example its purpose and audience within your organization. Leave this field blank to hide the link in the login and sign-up pages.'
```

Comment:
The URL for the About link on the Mattermost login and sign-up pages. If this field is empty, the About link is hidden from users.